canonical/store-components
# Add overflow to topic cards

## Done
Added properties to truncate topic card title and content

## QA
- Go to https://store-components-54.demos.haus/?path=/story/topiccard--truncate-title
- Check that the title is truncated
- Go to https://store-components-54.demos.haus/?path=/story/topiccard--truncate-content
- Check that the content is clamped at four lines
